### PR TITLE
Fix log collection

### PR DESCRIPTION
**Pull request by hithwen**
Fix log collection categories

### CODE REVIEW

**Comment by hithwen** [comment]
Pinging @kristianamitchellcontrastsecurity @jfrog/partner-engineering @gwaldo  @monganai and @brentm5 as integration maintainers

File: pihole/manifest.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
   ],
   "public_title": "Datadog-Pi-hole Integration",
   "categories": [
-    "network"
+    "network",
+    "log collection"
```
Comment (coignetp) [inline pihole/manifest.json:20]: Can you add a `log collection` section in the readme?

File: resin/manifest.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
   ],
   "public_title": "Datadog-Resin Integration",
   "categories": [
-    "web"
+    "web",
+    "log collection"
```
Comment (coignetp) [inline resin/manifest.json:20]: Can you add a `log collection` section in the readme?

**Review by coignetp — CHANGES_REQUESTED** [review]

**Review by monganai — APPROVED** [review]
LGTM for Pihole 👍

**Review by coignetp — APPROVED** [review]

**Pull request merged**